sift/client.py

```diff
@@ -313,6 +314,135 @@ def get_workflow_status(self, run_id, timeout=None):
+            limit: number of query results (decisions) to return [default: 100]
+            start_from: result set offset for use in pagination [default: 0]
+            abuse_types: csv of abuse_types to filter returned decisions by (optional)
+
+        Returns:
+            A sift.client.Response object containing array of decisions if call succeeded
+            Otherwise raises an ApiException
+        """
+
+        if timeout is None:
+            timeout = self.timeout
+
+        params = {}
+
+        if not isinstance(entity_type, self.UNICODE_STRING) or len(entity_type.strip()) == 0\
+                or entity_type not in {'user', 'order'}:
+            raise ApiException("entity_type must be one of {user, order}")
+
+        params['entity_type'] = entity_type
+
+        if limit:
+            params['limit'] = limit
+
+        if start_from:
+            params['from'] = start_from
+
+        if abuse_types:
+            params['abuse_types'] = abuse_types
+
+        try:
+            return Response(requests.get(self._get_decisions_url(self.account_id), params=params,
+                                         auth=requests.auth.HTTPBasicAuth(self.api_key, ''),
+                                         headers={'User-Agent': self._user_agent()}, timeout=timeout))
+
+        except requests.exceptions.RequestException as e:
+            raise ApiException(str(e))
+
+    def apply_user_decision(self, user_id, properties, timeout=None):
+        """Apply decision to user
+
+        Args:
+            user_id: id of user
+            apply_decision_request:
+                decision_id: decision to apply to user
+                source: {one of MANUAL_REVIEW | AUTOMATED_RULE | CHARGEBACK}
+                analyst: id or email, required if `source: MANUAL_REVIEW`
+                time: in millis when decision was applied
+        Returns
+            A sift.client.Response object if the call succeeded, else raises an ApiException
+        """
+
+        if timeout is None:
+            timeout = self.timeout
+
+        self.validate_apply_decision_request(properties, user_id, None)
+
+        try:
+            return Response(requests.post(
+                self._user_decisions_url(self.account_id, user_id),
+                data=json.dumps(properties),
+                auth=requests.auth.HTTPBasicAuth(self.api_key, ''),
+                headers={'Content-type': 'application/json',
+                         'Accept': '*/*',
+                         'User-Agent': self._user_agent()},
+                timeout=timeout))
+
+        except requests.exceptions.RequestException as e:
+            raise ApiException(str(e))
+
+    def apply_order_decision(self, user_id, order_id, properties, timeout=None):
+        """Apply decision to order
+
+        Args:
+            user_id: id of user
+            order_id: id of order
+            applyDecisionJson:
+                decision_id: decision to apply to user
```

Comment:
Have you decided that we will rely on the server to validate whether the required arguments are present?  That seems reasonable, though I think it would be worthwhile to say which are required explicitly.